src/client/datascience/activation.ts

```diff
@@ -48,27 +48,27 @@ export class Activation implements IExtensionSingleActivationService {
             return;
         }
         this.notebookOpened = true;
-        this.PreWarmDaemonPool().ignoreErrors();
+        void this.PreWarmDaemonPool();
         sendTelemetryEvent(Telemetry.OpenNotebookAll);
 
         if (!this.rawSupported.isSupported && this.extensionChecker.isPythonExtensionInstalled) {
             // Warm up our selected interpreter for the extension
-            this.jupyterInterpreterService.setInitialInterpreter().ignoreErrors();
+            void this.jupyterInterpreterService.setInitialInterpreter();
         }
     }
 
     private onDidChangeInterpreter() {
-        if (this.notebookOpened) {
+        if (this.notebookOpened && !this.rawSupported.isSupported && this.extensionChecker.isPythonExtensionInstalled) {
             // Warm up our selected interpreter for the extension
-            this.jupyterInterpreterService.setInitialInterpreter().ignoreErrors();
-            this.PreWarmDaemonPool().ignoreErrors();
+            void this.jupyterInterpreterService.setInitialInterpreter();
+            void this.PreWarmDaemonPool();
         }
     }
 
     @debounceAsync(500)
     @swallowExceptions('Failed to pre-warm daemon pool')
     private async PreWarmDaemonPool() {
-        if (!this.extensionChecker.isPythonExtensionActive) {
+        if (!this.extensionChecker.isPythonExtensionActive || this.rawSupported.isSupported) {
```

Comment:
I'm confused by this? Why not prewarm the daemon pool when using raw? We still use daemons for raw.

Reply:
Here we're attempting to pre-warm the daemon pool against the python env that starts jupyter.
When using raw, we pre-warm against the python env that's associated with the kernel.

Its possible that the kernel selected is not the same as the python env used to start jupyter, 
If using non-raw, then it makes sense

Reply:
I guess the problem then is nobody knows (at that point in the code) why raw isn't allowed. Can you put a comment explaining why?